tests/integration/census/test_census_communal_submission_data.py

```diff
@@ -0,0 +1,157 @@
+
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestCensusCommunalSubmissionData(IntegrationTestCase):
+
+    def test_census_communal_data_matches_census_communal(self):
+        self.complete_survey('census', 'communal')
+
+        # Only verifying 'data'
+        actual_downstream_data = self.dumpSubmission()['submission']['data']
+        expected_downstream_data = self.get_expected_submission_data()
+
+        self.assertCountEqual(actual_downstream_data, expected_downstream_data)
```

Comment:
This is only testing it has the same number of answers and not that the data in actually what was entered?